modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -564,4 +565,17 @@ describe('browser.tinymce.core.UndoManager', () => {
     editor.execCommand('mceFocus');
     assert.isFalse(editor.undoManager.hasUndo());
   });
+
+  it('TINY-7663: Undo when first element is contenteditable="false" should restore correct cursor location', () => {
```

Comment:
Can we add a test where the cursor is also in the fake caret when the undo bookmark is created please? or does that already exist in this file already (I couldn't see one)?